lib-database/src/commonMain/kotlin/com/ustadmobile/core/db/ext/UmAppDatabaseMigrations.kt

```diff
@@ -1692,15 +1693,23 @@ val MIGRATION_206_207 = DoorMigrationStatementList(206, 207) { db ->
     }
 }
 
+
 val MIGRATION_207_208 = DoorMigrationStatementList(207, 208) { db ->
+
```

Comment:
This migration number needs to be bumped, and previous 207-208 migration should be here.